### PR TITLE
fix(curriculum): Remove end of line chars after code blocks

### PR DESCRIPTION
**Pull request by RandellDawson**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

This PR fixes a bug on Crowdin that does not allow us to hide these single line code blocks which have characters (included extra white space) at the end of them.

### CODE REVIEW

**Review by naomi-lgbt — DISMISSED** [review]
Looks good to me.

File: curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-priority-queue-class.md [hunk 1]
```diff
@@ -12,11 +12,15 @@ In this challenge you will be creating a Priority Queue. A Priority Queue is a s
...
+```js
+[['kitten', 2], ['dog', 2], ['rabbit', 2]]
+```
 
 Here the second value (an integer) represents item priority. If we enqueue `['human', 1]` with a priority of `1` (assuming lower priorities are given precedence) it would then be the first item to be dequeued. The collection would look like this:
 
-`[['human', 1], ['kitten', 2], ['dog', 2], ['rabbit', 2]]`.
+```js
+[['human', 1], ['kitten', 2], ['dog', 2], ['rabbit', 2]]
+````
```
Comment (raisedadead) [inline curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-priority-queue-class.md:23]: Are the 4 backticks intentional?

Comment (RandellDawson) [reply curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-priority-queue-class.md:23]: No, that was not intentional.  I will push up another commit to remove.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Review by naomi-lgbt — APPROVED** [review]
Looked good to me before, looks great to me now 🙃

**Pull request merged**